domain-service/credit-application/src/main/java/at/deloittedigital/core_banking/credit/datasource/RemoteCreditDataService.java

```diff
@@ -7,17 +7,27 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
 import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestTemplate;
-import org.springframework.web.util.UriComponentsBuilder;
 
 @Service
 @Log4j2
 @ConditionalOnProperty(name = "core-banking.credit.datasource.mode", havingValue = "remote")
 public class RemoteCreditDataService implements CreditDataService {
 
+    // todo hardcoded graphql request for testing
+    private static final String TEST_QUERY =
```

Comment:
sorry, did you mean creating DTO's for the queries in the todo?

Reply:
Yes, something that makes sense as a graphql API client. In this PR my goal was to integrate domain svc <-> data svc with some sample query that gets client info, and for that I used this hardcoded string query.
In the future I'd like to create something more OO for it that can be used to handle the various queries we need for the credit checks. I added AB#2337 for it.